mne/viz/tests/test_evoked.py

```diff
@@ -133,7 +133,7 @@ def test_plot_evoked():
 
     # test `gfp='only'`: GFP (EEG) and RMS (MEG)
     fig, ax = plt.subplots(3)
-    evoked.plot(gfp='only', time_unit='s', axes=ax)
+    evoked.plot(gfp='only', time_unit='s', axes=ax, spatial_colors=False)
```

Comment:
the fact that you need to add this False in many places is a code smell that suggests it can break users code. Can you explain why you needed these?

Reply:
In some places, such as this one, the addition was unnecessary, and the lack of it did not break the test; I thought I should return the tests to the original states (since the argument was 'False' by default). In some places evoked.plot() is followed by _get_amplitudes(fig), which throws a VisibleDeprecationWarning caused by trying to create np. array from lists with different lengths. In one other line, there is an assertion about the highlighted areas of the plot, which probably fails spatial_colors arguments is 'True' rather than 'False'. I think because the tests were written for the 'False' default argument. I am not sure if this would break the user code.

Reply:
ok it makes more sense to me now 🙏 

@HuseyinOrkun can I ask you to rebase this branch on current main branch and fix the conflicts to make the CIs green? 🙏

Reply:
I have a question, which is better, not changing the tests that do not break without the additional spatial_colors='False' argument or adding spatial_colors='False' to preserve the original arguments in those tests? Thank you very much for your feedback :)